tmp/.htmlproofer/cache.log

```diff
@@ -0,0 +1 @@
+{"https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"https://github.com/duraspace/pcdm/wiki":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"https://github.com/islandora-interest-groups/Islandora-Metadata-Interest-Group/wiki":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"https://github.com/duraspace/pcdm/wiki/Presentations":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"http://projecthydra.org":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"https://github.com/duraspace/pcdm":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"https://github.com/duraspace/pcdm/issues":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"https://groups.google.com/d/forum/pcdm":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"http://gmpg.org/xfn/11":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"http://islandora.ca":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"https://wiki.duraspace.org/display/hydra/Hydra+Metadata+Interest+Group":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"http://fcrepo.org":{"time":"2016-03-28 11:21:56 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"http://pcdm.org/use":{"time":"2016-03-28 11:24:39 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"http://pcdm.org/rights":{"time":"2016-03-28 11:24:39 -0400","filenames":["./_site/index.html"],"status":200,"message":""},"http://pcdm.org/models":{"time":"2016-03-28 11:24:39 -0400","filenames":["./_site/index.html"],"status":200,"message":""}}
```

Comment:
Should clarify, in iiif.io-land we've had a lot of link checks fail randomly on Travis and it seems to just be due to limits of the build environment.